grokability/snipe-it
# Updated Livewire to v3

# Description

This PR updates Livewire from v2 to v3.

I started with the [automated upgrader](https://livewire.laravel.com/docs/upgrading#automated-upgrade-tool) before moving on to the manual changes that needed to be made.

Notables:
- The standard namespace for Livewire in v3 is `App\Livewire` so our existing components have been moved there.
- Alpine has been removed from the app's `package.json` since it is bundled with [Livewire now](https://livewire.laravel.com/docs/installation#manually-including-livewires-frontend-assets). Since it is used in the new label engine I added `@livewireScripts` to that page. Any future pages that use Alpine but not Livewire will need to have that added.
- The current state of the importer requires `legacy_model_binding` to be turned on but I would like to change that back to the [default](https://livewire.laravel.com/docs/upgrading#eloquent-model-binding) in the future.
- There are some `wire:model.live`s that can probably have `live` removed but that's a small change that can happen in the future.

To test this make sure you:
- `composer install`
- `nvm use`
- `npm run dev`

Please test thoroughly to double-check I didn't miss something.

The pages affected:
- `/account/api`
- `/admin/oauth`
- `/admin/slack`
- `/categories/create` and `/categories/{id}/edit`
- `/import`
- `/models/create` and `/models/{id}/edit`

## Type of change

- [x] Library update
